akka-projection-grpc/src/it/scala/akka/projection/grpc/IntegrationSpec.scala

```diff
@@ -173,8 +183,16 @@ class IntegrationSpec(testContainerConf: TestContainerConf)
         EventProducerSource(source.entityType, source.streamId, transformation, EventProducerSettings(system)))
       .toSet
 
+    val authInterceptor = new EventProducerInterceptor {
+      def intercept(streamId: String, requestMetadata: Metadata): Future[Done] = {
```

Comment:
no need to be able to return new Metadata, instead of Done?

Reply:
We never actually look at the metadata where we call `intercept` in the event producer service impl.